scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -214,9 +212,31 @@ public static BuildTargetResult CompileStage2(BuildTargetContext c)
                 }
             }
 
+            CleanOutputDir(Path.Combine(Dirs.Stage2, "sdk"));
```

Comment:
pubs will still be in the sdk compilation folder right? Could copy them over.

I guess we could change packaging not to pick up stage2 and create a special packaging stage2 as an alternative.